src/components/TodoApp.jsx

```diff
@@ -0,0 +1,18 @@
+import { Routes, Route } from 'react-router-dom';
+
+import Home from '@/routes/Home';
+import NotMatch from '@/routes/NotMatch';
+import Layout from '@/components/Layout';
+
+
+const TodoApp = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route index element={<Home />} />
+        <Route path="*" element={<NotMatch />} />
+      </Route>
+    </Routes>
```

Comment:
### Optional Suggestions:
- Following rules "YAGNI", there are some unnecessary modules, please remove them if you don't need them.

Reply:
Thank you for your Review @khanh-devos and thank you for sharing the suggested changes to improve the project. I have removed the unused modules. Thanks